app/models/manageiq/providers/openshift/container_manager/refresher.rb

```diff
@@ -3,9 +3,18 @@ module Openshift
     class ContainerManager::Refresher < ManageIQ::Providers::BaseManager::Refresher
       include ::EmsRefresh::Refreshers::EmsRefresherMixin
 
+      KUBERNETES_EMS_TYPE = ManageIQ::Providers::Kubernetes::ContainerManager.ems_type
```

Comment:
Why are we creating this constant?  The string is frozen and memo-ized already [here](https://github.com/ManageIQ/manageiq/blob/master/app/models/manageiq/providers/openshift/container_manager.rb#L33-L35)